Sources/SwiftProtobuf/MessageExtension.swift

```diff
@@ -18,7 +18,7 @@
 // TODO: `AnyMessageExtension` should require `Sendable` but we cannot do so yet without possibly breaking compatibility.
```

Comment:
This comment can go away, it's left from the 1.x branch.

Reply:
Done